src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -221,7 +221,7 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
       if (response.requestSucceeded) {
         reload();
         onCloseHandler();
-        if (testCompleted && !readOnly) {
+        if ((testCompleted || record.testType === 'Cut') && !readOnly) {
```

Comment:
In cut test readOnly will always be false (we can't mark it as complete), so if we are editing the cut test and not changing any value we will see the viability message again. Not sure if that's correct

Reply:
good catch, i will update this to only check when the value is edited

Reply:
updated and tested